libvast/test/schema.cpp

```diff
@@ -105,14 +104,12 @@ TEST(merging) {
 
 TEST(serialization) {
   schema sch;
-  auto t = record_type{
-    {"s1", string_type{}},
-    {"d1", real_type{}},
-    {"c", count_type{}.attributes({{"skip"}})},
-    {"i", integer_type{}},
-    {"s2", string_type{}},
-    {"d2", real_type{}}
-  };
+  auto t = record_type{{"s1", string_type{}},
+                       {"d1", real_type{}},
+                       {"c", count_type{}.attributes({{"skip"}})},
+                       {"i", integer_type{}},
+                       {"s2", string_type{}},
+                       {"d2", real_type{}}};
```

Comment:
For these, if you add a trailing comma you get better formatting behavior of clang-format. Not a must, just improves the readability.

```suggestion
                       {"d2", real_type{}},};
```